libgstc/libgstc.c

```diff
@@ -392,6 +392,24 @@ gstc_pipeline_stop (GstClient * client, const char *pipeline_name)
   return gstc_cmd_change_state (client, pipeline_name, state);
 }
 
+GstcStatus
+gstc_pipeline_get_graph (GstClient * client, const char *pipeline_name, char **response)
+{
+
+  GstcStatus ret;
+
+  char uri[sizeof("/pipelines/%s/graph") + sizeof(pipeline_name)];
+
+  gstc_assert_and_ret_val (NULL != client, GSTC_NULL_ARGUMENT);
+  gstc_assert_and_ret_val (NULL != pipeline_name, GSTC_NULL_ARGUMENT);
+  gstc_assert_and_ret_val (NULL != response, GSTC_NULL_ARGUMENT);
+
+  sprintf(uri, "/pipelines/%s/graph",pipeline_name);
```

Comment:
This is unsafe code, follow the pattern in the rest of the document! This will crash if uri doesn't have enough space to fit the desired text. The other methods use `asprintf` which reserves the required memory for you.